includes/data.php

```diff
@@ -1497,13 +1498,9 @@ function pods_evaluate_tags( $tags, $sanitize = false ) {
 		return $tags;
 	}
 
-	$callback = 'pods_evaluate_tag';
-
-	if ( true === $sanitize ) {
-		$callback = 'pods_evaluate_tag_sanitized';
-	}
-
-	return preg_replace_callback( '/({@(.*?)})/m', $callback, (string) $tags );
+	return preg_replace_callback( '/({@(.*?)})/m', function ( $tag ) use ( $sanitize, $fallback ) {
+		return pods_evaluate_tag( $tag, $sanitize, $fallback );
+	}, (string) $tags );
```

Comment:
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call
* :no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 94872091deaa30cc31a470869522392c</sub>